src/Finder/SourceFilesFinder.php

```diff
@@ -80,7 +80,7 @@ public function getSourceFiles(string $filter = ''): Finder
             return $this;
         }
 
-        $filters = explode(',', $filter);
+        $filters = array_filter(explode(',', $filter));
```

Comment:
Hopefully no one will ever need to run Infection over a file named `"0"`. What do you think?

Reply:
Considering their PHP Install would have to be very specific to do that, I don't think it'll be a problem.